stage-file-proxy.php

```diff
@@ -128,26 +134,21 @@ function sfp_dispatch() {
 			} else {
 				sfp_serve_requested_file( $basefile );
 			}
-		} elseif ( 'lorempixel' === $mode ) {
-			$width = $doing_resize && ! empty( $resize['width'] ) ? $resize['width'] : 800;
-			$height = $doing_resize && ! empty( $resize['height'] ) ? $resize['height'] : 600;
-			header( 'Location: http://lorempixel.com/' . $resize['width'] . '/' . $resize['height'] );
-			exit;
```

Comment:
if we want to keep this functionality, it appears https://picsum.photos/ could be a drop in replacement